models/models_methods.go

```diff
@@ -779,7 +779,7 @@ func MergeStepWith(step, otherStep stepmanModels.StepModel) (stepmanModels.StepM
 // --- WorkflowIDData
 
 // GetWorkflowIDFromListItemModel ...
-func GetWorkflowIDFromListItemModel(workflowListItem WorkflowListItemModel) (string, error) {
+func GetWorkflowIDFromListItemModel(workflowListItem StageWorkflowListItemModel) (string, error) {
 	if len(workflowListItem) > 1 {
 		return "", errors.New("WorkflowListItem contains more than 1 key-value pair")
```

Comment:
NIT: shouldn't we use the `StageWorkflowListItem` word in the error message?

Reply:
I have update the error messages in this and the `GetStageIDFromListItemModel` too.